app/controllers/api/v1/services_controller.rb

```diff
@@ -0,0 +1,31 @@
+class Api::V1::ServicesController < ApplicationController
+  def index
+    services = Service.all.order('created_at DESC')
+    render status: 200, json: { services: services }.to_json
+  end
+
+  def create
+    @service = Service.new(service_params)
+
+    if @service.save
+      render status: 201, json: { message: 'Service has been created' }.to_json
+    else
+      render status: 500, json: { message: 'Service creation has failed' }.to_json
```

Comment:
Nice adding the status codes! I didn't know how to do it, now I'll just copy you 😁